.github/actions/pr-checklist/index.js

```diff
@@ -1,338 +0,0 @@
-      console.log(
-        'The event that triggered this action was not a pull request, skipping.'
-      );
-      return;
-    }
-    
-    const prTitle = context.payload.pull_request.title;
-    const prDescription = context.payload.pull_request.body;
-
-    const client = github.getOctokit(
-        core.getInput('repo-token', {required: true})
-    );
-
-    const files = (await client.paginate(client.rest.pulls.listFiles, {
-        owner: context.issue.owner,
-        repo: context.issue.repo,
-        pull_number: context.issue.number,
-    })).data;
-    const comments = (await client.rest.issues.listComments({
-        owner: context.issue.owner,
-        repo: context.issue.repo,
-        issue_number: context.issue.number,
-    })).data;
-    const commits = (await client.rest.pulls.listCommits({
-        owner: context.issue.owner,
-        repo: context.issue.repo,
-        pull_number: context.issue.number,
-    })).data;
-
-    const theComment = comments.find(c => c.body.includes(commentHeader));
-    
-    const errors = [];
-
-    if (theComment) {
-        // comment already exists, update it appropriately
-        const existingBody = theComment.body;
-        if (!bugRefsVerified(getCommentSection(BUG_REFS, existingBody))) {
-            errors.push("Bug references list has not been confirmed.")
-        }
-        if (!testUpdatesVerified(getCommentSection(TEST_UPDATES, existingBody))) {
-            errors.push("Test updates section has not been filled out.")
-        }
-        if (!docsVerified(getCommentSection(DOCS, existingBody))) {
-            errors.push("Documentation section has not been filled out.")
-        }
-        if (!extraReviewsVerified(getCommentSection(EXTRA_REVIEWS, existingBody))) {
-            errors.push("Extra reviews section has not been filled out.")
-        }
-        const newBody = createComment({
-            prTitle,
-            prDescription,
-            commits,
-            files,
-            existingBody,
-        })
-        if (existingBody !== newBody) {
-            client.rest.issues.updateComment({
-                owner: context.issue.owner,
-                repo: context.issue.repo,
-                comment_id: theComment.id,
-                body: newBody,
-            });
-        }
-    } else {
-        // first run, comment doesn't exist yet
-        const newBody = createComment({
-            prTitle,
-            prDescription,
-            commits,
-            files,
-            existingBody: null,
-        })
-        client.rest.issues.createComment({
-            owner: context.issue.owner,
-            repo: context.issue.repo,
-            issue_number: context.issue.number,
-            body: newBody,
-        });
-        client.rest.reactions.createForIssue({
-            owner: context.issue.owner,
-            repo: context.issue.repo,
-            issue_number: context.issue.number,
-            content: "eyes"
```

Comment:
We'll have to put this code in a separate action as it's actually very important for the repo
But that can be done later